meson.build

```diff
@@ -18,9 +18,12 @@ conf.set('MICRO_VERSION', fwupd_micro_version)
 conf.set_quoted('PACKAGE_VERSION', fwupd_version)
 
 # get source version, falling back to package version
+fs = import('fs')
 git = find_program('git', required: false)
 tag = false
-if git.found()
+if fs.is_file('VERSION')
+  source_version = fs.read('VERSION', encoding: 'utf-8').strip()
```

Comment:
not a huge fan; how come you can't use git here?

Reply:
I think it doesn't have .git at this point. I'll double check.

Reply:
seems it works actually, I must have had some transient problem.